### PR TITLE
Python Flask template

### PR DESCRIPTION
**Pull request by bmwilly**

### CODE REVIEW

**Review by ian-contiamo — APPROVED** [review]

File: template/python-flask/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY index.py .
+COPY handler.py .
+
+ENV fprocess="python index.py"
+ENV cgi_headers="true"
+ENV mode="http"
+ENV upstream_url="http://127.0.0.1:5000"
```
Comment (trusch) [inline template/python-flask/Dockerfile:21]: What is that for?

Comment (bmwilly) [reply template/python-flask/Dockerfile:21]: I'm not sure, but it doesn't seem to work without it: https://github.com/openfaas-incubator/python-flask-template/blob/master/template/python3-flask/Dockerfile#L31

**Review by trusch — COMMENTED** [review]
If we use flask which is itself a Webserver, why use fwatchdog? I think watchdog expects the fprocess to read from stdin.

Imo you should consider listening on port 8080 + write the openfaas specific lockfile when flask is up and running.
If I read this correctly, I doubt it would work as it is.

Edit: forget about this, I missed the mode=http part

Edit2: in my opinion the second http server layer (fwatchdog) is useless here, but this is how it is supposed to work

File: template/python-flask/Dockerfile (hunk 1 above)
Comment (LucasRoesler) [reply template/python-flask/Dockerfile:21]: That is used by the watchdog

**Comment by bmwilly** [comment]
@trusch using this template, I was able to create successful dummy and [Telekom](https://github.com/contiamo/telekom-challenge/tree/master/functions/predict) functions.

**Review by LucasRoesler — APPROVED** [review]

**Pull request merged**